src/sys/socket/mod.rs

```diff
@@ -748,6 +748,14 @@ pub enum ControlMessageOwned {
     #[cfg(feature = "net")]
     #[cfg_attr(docsrs, doc(cfg(feature = "net")))]
     Ipv6PacketInfo(libc::in6_pktinfo),
+    #[cfg(any(linux_android, target_os = "freebsd", target_os = "ios", target_os = "macos"))]
```

Comment:
Here and elsewhere, unless you'rre very sure that this doesn't work on tvos or watchos, just use `apple_targets`.
```suggestion
    #[cfg(any(apple_targets, linux_android, target_os = "freebsd"))]
```